client/my-sites/checkout/checkout-thank-you/jetpack-standalone-activation-instructions.tsx

```diff
@@ -0,0 +1,60 @@
+import { useTranslate } from 'i18n-calypso';
+import { FC } from 'react';
+import ExternalLink from 'calypso/components/external-link';
+import { SelectorProduct } from 'calypso/my-sites/plans/jetpack-plans/types';
+
+interface Props {
+	product: SelectorProduct;
+}
+
+const JetpackStandaloneActivationInstructions: FC< Props > = ( { product } ) => {
+	const translate = useTranslate();
+	return (
+		<>
+			<ul className="licensing-thank-you-manual-activation-instructions__list">
+				<li className="licensing-thank-you-manual-activation-instructions__list-item">
+					<span className="licensing-thank-you-manual-activation-instructions__step-number">1</span>
+					<span>{ translate( 'Login to an existing Wordpress site as an administrator.' ) }</span>
+				</li>
+
+				<li className="licensing-thank-you-manual-activation-instructions__list-item">
+					<span className="licensing-thank-you-manual-activation-instructions__step-number">2</span>
+					<span>
+						{ translate(
+							'Go to {{strong}}Plugins > add New{{/strong}} in the admin menu on the left hand side.',
+							{
+								components: { strong: <strong /> },
+							}
+						) }
+					</span>
+				</li>
+
+				<li className="licensing-thank-you-manual-activation-instructions__list-item">
+					<span className="licensing-thank-you-manual-activation-instructions__step-number">3</span>
+					<span>
+						{ translate(
+							'Search for {{strong}}Jetpack %(pluginName)s{{/strong}}, install and activate.',
+							{
+								components: { strong: <strong /> },
+								args: { pluginName: product.shortName },
+							}
+						) }
```

Comment:
Can we make the plugin here a hyperlink to plugin URL on wp.org plugin directory?